pkg/operator/ceph/config/defaults.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package config provides default configurations which Rook will set in Ceph clusters.
+package config
+
+import "github.com/coreos/pkg/capnslog"
+
+// DefaultCentralizedConfigs returns the default configuration options Rook will set in Ceph's
+// centralized config store. If the version of Ceph does not support the centralized config store,
+// these will be set in a shared config file instead.
+func DefaultCentralizedConfigs(rookLogLevel capnslog.LogLevel) *Config {
+	cephLogLevel := logLevelToCephLogLevel(rookLogLevel)
+
+	c := NewConfig()
+	c.Section("global").
+		Set("log to stderr", "true").
+		Set("err to stderr", "true").
+		Set("log file", "").
+		Set("mon cluster log file", "").
+		Set("log stderr prefix", "\"debug \"").
+		// ^ help differentiate debug text from audit text, and the space after 'debug' is critical
+		Set("debug default", cephLogLevel).
+		Set("debug rados", cephLogLevel).
+		Set("debug mon", cephLogLevel).
+		Set("debug osd", cephLogLevel).
+		Set("debug bluestore", cephLogLevel).
+		Set("debug filestore", cephLogLevel).
+		Set("debug journal", cephLogLevel).
+		Set("debug leveldb", cephLogLevel).
+		//
+		Set("mon allow pool delete", "true").
+		Set("fatal signal handlers", "false")
+
+	return c
+}
+
+// convert a Rook log level to a corresponding Ceph log level
+func logLevelToCephLogLevel(logLevel capnslog.LogLevel) string {
```

Comment:
if we remove the logging options, we won't need this conversion, although saving it for a separate PR would be fine too